public/linkedhashes.php

```diff
@@ -41,7 +41,13 @@
              "<a href='https://docs.retroachievements.org/Game-Identification/'>here</a>." .
              "</b></p>";
 
-        echo "<p class='mt-4 mb-1'>There are currently <span class='font-bold'>" . count($hashes) . "</span> supported game file hashes registered for this game.</p>";
+        echo "<div class='mt-4 mb-1'>";
+        if ($permissions >= Permissions::Developer) {
+            echo "<a href='/managehashes.php?g=$gameID'>Manage Hashes</a>";
+        }
```

Comment:
For consistency, I would recommend putting the link behind a `Dev \/` collapsible section. Alternately, maybe it could be appended as subtext in the line: `There are currently X supported game file hashes registered for this game. (Manage Hashes)`

Something about where it's sitting just doesn't feel right to me.

Reply:
I like the devbox idea the best of the two, it feels more consistent with other functionality on the site. I've made this change in https://github.com/RetroAchievements/RAWeb/pull/1436/commits/1ebf36de1f55f42f149f7063728f2d1c61b12887.